docs/contributor/pocs/ginkgo-to-go-testing/ginkgo-to-go-testing.md

```diff
@@ -0,0 +1,19 @@
+# Migrate Ginkgo tests to Go testing PoC
+
+This proof of concept demonstrates how to migrate Ginkgo tests to Go's built-in testing framework. The goal is to convert Ginkgo-style tests into the plain Go testing style while maintaining the same functionality and structure.
+
+## Built-in testing E2E test example
+
+[Service Name Enrichment](./service_name_test.go)
+
+## Gomega
+
+Gomega is a matcher library for Go that provides expressive assertions. Even though it is often used together with Ginkgo, it can be also used with the built-in `testing` package to write tests in a more readable way. It means that we can keep our Gomega matchers while migrating from Ginkgo to Go's testing framework. Note that in every test func we must call `gomega.RegisterTestingT(t)` to register the testing.T instance with Gomega.
```

Comment:
```suggestion
Gomega is a matcher library for Go that provides expressive assertions. Even though it is often used together with Ginkgo, it can be also used with the built-in `testing` package to write tests in a more readable way. It means that we can keep our Gomega matchers while migrating from Ginkgo to Go's testing framework. Note that in every test func, we must call `gomega.RegisterTestingT(t)` to register the testing.T instance with Gomega.
```

I trust "testing.T" is not a typo, but it looks weird to me :D

Reply:
It's not :) https://pkg.go.dev/testing#T